automation/jleechanorg_pr_automation/check_codex_comment.py

```
@@ -55,7 +55,13 @@ def decide(marker_prefix: str, marker_suffix: str) -> Tuple[str, str]:
         if end_index == -1:
             continue
 
-        marker_sha = body[start_index:end_index].strip()
+        marker_content = body[start_index:end_index].strip()
+        # Handle new format: SHA:cli -> extract just SHA
+        # Also handles old format: SHA (no colon)
+        if ":" in marker_content:
+            marker_sha = marker_content.split(":")[0]
+        else:
+            marker_sha = marker_content
```

Ruff (0.14.11)

Use ternary operator `marker_sha = marker_content.split(":")[0] if ":" in marker_content else marker_content` instead of `if`-`else`-block

Replace `if`-`else`-block with `marker_sha = marker_content.split(":")[0] if ":" in marker_content else marker_content`

(SIM108)